Core/Link.swift

```diff
@@ -28,12 +28,14 @@ public class Link: NSObject, NSCoding {
     private struct NSCodingKeys {
         static let title = "title"
         static let url = "url"
+        static let localPath = "localPath"
     }
     
     static let appUrls = AppUrls()
 
     public let title: String?
     public let url: URL
+    public let localPath: URL?
```

Comment:
My suggestion is to rename it to: `localFileURL` instead - since when `path` keyword is used in the iOS APIs it is a String.